qmail-smtpd.c

```diff
@@ -235,6 +269,9 @@ void smtp_rset(arg) char *arg;
 void smtp_mail(arg) char *arg;
 {
   if (!addrparse(arg)) { err_syntax(); return; }
+#ifdef SMTPUTF8
+  if (flagutf8) mailfrom_parms(arg);
+#endif
```

Comment:
Boolean checks are cheap

Reply:
same as previous comment. Let us remove flagutf8

Reply:
I was trying to say let's keep `flagutf8` and get rid of the `#ifdef`

Reply:
Ok